HumanControls/Page.h

```diff
@@ -19,13 +19,13 @@ class Page : DisplayInterface
         DEBUG_PAGE = 4
     };
 
-    Page(bool isActive, bool applyHang, unsigned int downArrow, unsigned int upArrow,
-         unsigned int robotBatChar, unsigned int controllerBatChar, Page::PageType name);
+    Page(bool isActive, bool applyHang, Page::PageType name);
     void cleanUp(DisplayController &display);
     virtual void paint(DisplayController &display, bool isActivated, TShirtCannonPayload &payload) = 0;
     virtual void clockwise(TShirtCannonPayload &payload) = 0;
     virtual void counterClockwise(TShirtCannonPayload &payload) = 0;
     virtual int rangeFilter(int value) = 0;
+    virtual bool hasUpdated(TShirtCannonPayload &payload) = 0;
```

Comment:
I notice you always return `true` for this function. If you have plans to implement this functionality in the future it can stay, if not this function should be removed.